src/game/models/monsters.js

```diff
@@ -500,7 +500,7 @@ t . Fire Emblem
     prizes: {
       exp: 110,
       items: ['Molten Feather'],
-      equips: ['molten_gauntlets', 'flame'],
+      equips: ['flame', 'magma', 'lava', { id: 'molten_gauntlets', weight: 2 }],
```

Comment:
It's "1" by default, just like it did until now.
If you prefer I could use the new object format for all drops. I wouldn't mind changing it.